stdlib/contrib/bonitoo-io/victorops/victorops_test.go

```diff
@@ -0,0 +1,246 @@
+package victorops_test
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/influxdata/flux"
+	_ "github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/dependencies/dependenciestest"
+	_ "github.com/influxdata/flux/fluxinit/static"
+	"github.com/influxdata/flux/lang"
+	"github.com/influxdata/flux/memory"
+	"github.com/influxdata/flux/runtime"
+)
+
+func TestVictorOps(t *testing.T) {
+	ctx := dependenciestest.Default().Inject(context.Background())
+	_, scope, err := runtime.Eval(ctx, `
```

Comment:
You can just remove `scope` here. Then you don't need to do the fake assignment on line 53.

```go
_, _, err := runtime.Eval(...)
```